deployment/clouddeploy/gke-workers/environments/oss-vdb-test/staging-test.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: staging-test
+spec:
+  schedule: "50 9 * * *"
```

Comment:
Was there any specific reason to run this at exactly 9:50 AM?

Reply:
Querying vulnerabilities takes around 10 minutes to finish, so I wanted it to complete around 11 AM to allow us enough time to debug if any errors occur.